process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessJsonSchemaDocumentInstance.java

```diff
@@ -38,6 +40,14 @@
     @Column(name = "process_name", columnDefinition = "VARCHAR(255)")
     private String processName;
 
+    @Formula("( " +
```

Comment:
Iam missing the integration test that asserts this value is set please update existing or add new testcase.

Reply:
I have switched to a better method for querying statusses and expanded an integration test with some assertions